examples/gnrc_networking/udp.c

```diff
@@ -30,10 +30,15 @@
 #include "net/gnrc/pktdump.h"
 #include "timex.h"
 #include "utlist.h"
+#if IS_USED(MODULE_ZTIMER_MSEC)
+#include "ztimer.h"
+#else
 #include "xtimer.h"
+#endif
 
-static gnrc_netreg_entry_t server = GNRC_NETREG_ENTRY_INIT_PID(GNRC_NETREG_DEMUX_CTX_ALL,
-                                                               KERNEL_PID_UNDEF);
+static gnrc_netreg_entry_t server =
+                        GNRC_NETREG_ENTRY_INIT_PID(GNRC_NETREG_DEMUX_CTX_ALL,
+                                                   KERNEL_PID_UNDEF);
 
 
```

Comment:
The line above seems to be a style fix, so care to add another one and remove one of these two lines? See also [static tests](https://github.com/RIOT-OS/RIOT/pull/16343/checks?check_run_id=2800159909#step:5:20).